client/a8c-for-agencies/sections/client/primary/checkout-v2/index.tsx

```diff
@@ -142,7 +142,7 @@ function ClientCheckoutContent() {
 		<div className="client-checkout-v2">
 			<CheckoutMain
 				sitelessCheckoutType="a4a"
-				redirectTo="/client/subscriptions"
+				redirectTo={ window.location.origin + '/client/subscriptions' }
```

Comment:
As the checkout process transitions to the Dotom domain, we must pass the origin information. This will enable the Pending page from the Dotcom domain to properly redirect users back to A4A domain.